src/isIsogram.test.js

```diff
@@ -6,4 +6,31 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`True with 'isog'`, () => {
+    expect(isIsogram('isog'))
+      .toBeTruthy();
+  });
+
+  it(`False with 'isogg'`, () => {
```

Comment:
Consider providing more descriptive test case names. For example, instead of `False with 'isogg'`, you could use `returns false for a non-isogram word`.